src/util/ChartUtils.ts

```diff
@@ -385,7 +386,7 @@ export const appendOffsetOfLegend = (offset: any, items: Array<FormattedGraphica
     const box = legendBox || {};
     const { align, verticalAlign, layout } = legendProps;
 
-    if ((layout === 'vertical' || (layout === 'horizontal' && verticalAlign === 'center')) && isNumber(offset[align])) {
+    if ((layout === 'vertical' || (layout === 'horizontal' && verticalAlign === 'middle')) && isNumber(offset[align])) {
```

Comment:
As this is a clear bug fix, I think we are good to keep this change. Center was never an option :)

Reply:
+1, nice find